lib/patreon/api.rb

```diff
@@ -33,16 +33,17 @@ def fetch_page_of_pledges(campaign_id, page_size, cursor=nil, includes=nil, fiel
     private
 
     def get_json(suffix)
-      http = Net::HTTP.new("api.patreon.com", 443)
+      http = Net::HTTP.new("www.patreon.com", 443)
       http.use_ssl = true
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
this is a little terrifying

Reply:
why

Reply:
It seems like it would be nice if we validated SSL certs

Reply:
Ohhh... fascinating, yeah. Uh... I don't think we do that for _any_ of our clients right now, but we can add doing so to our backlog.